docs/conf.py

```diff
@@ -25,6 +25,8 @@ def _get_version():
     return '.'.join(version_parts)
 
 
+nitpicky = True
```

Comment:
Why don't you keep it close to it's "ignore"? I normally even make a comment-delimited subsection dedicated to strictness settings that also includes "default_role".

Reply:
I had actually forgotten about the `nitpick_ignore` setting at the bottom of the file (and also noticed this line is a duplicate). I'm working through fixing those now.